tests/FakeItEasy.Tests.ruleset

```diff
@@ -23,6 +25,7 @@
     <Rule Id="CA1709" Action="None" />
     <Rule Id="CA1716" Action="None" />
     <Rule Id="CA1720" Action="None" />
+    <Rule Id="CA1724" Action="None" />

```

Comment:
"Type names should not match namespaces". Doesn't matter in tests. Violated by `Generic`, which matches `System.Collection.Generic`